SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestPropertyReplication/SpatialTestPropertyReplication.cpp

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialTestPropertyReplication.h"
+#include "Kismet/GameplayStatics.h"
+#include "ReplicatedTestActor.h"
+
+/**
+ * This tests that an Actor can replicate a property across the network during play.
+ * This test contains 1 Server and 2 Client workers.
+ *
+ * The flow is as follows:
+ * - Setup:
+ *  - The Server spawns one ReplicatedTestActor.
+ * - Test:
+ *  - Both Clients check that they can see exactly 1 ReplicatedTestActor.
+ *  - The Server changes the ReplicatedProperty of the ReplicatedTestActor from "0" to "99".
+ *  - Both Clients check that the ReplicatedProperty is now set to "99".
+ * - Clean-up:
+ *  - ReplicatedTestActor is destroyed using the RegisterAutoDestroyActor helper function.
+ */
+
+ASpatialTestPropertyReplication::ASpatialTestPropertyReplication()
+	: Super()
+{
+	Author = "Ollie Balaam";
+	Description = TEXT("This tests that an Actor can replicate a property across the network during play.");
+}
+
+void ASpatialTestPropertyReplication::PrepareTest()
+{
+	Super::PrepareTest();
+
+	AddStep(TEXT("The Server spawns one ReplicatedTestActor"), FWorkerDefinition::Server(1), nullptr, [this]() {
+		TestActor =
+			GetWorld()->SpawnActor<AReplicatedTestActor>(FVector(100.0f, 100.0f, 80.0f), FRotator::ZeroRotator, FActorSpawnParameters());
```

Comment:
Any reason for 100, 100, 80? Usually I would just do something at 0, 0.

Reply:
> Any reason for 100, 100, 80? Usually I would just do something at 0, 0.

I think I inherited this from another test I pinched from. I can set it to `0.0f, 0.0f, 0.0f` is prefereable?